src/main.rs

```diff
@@ -206,12 +207,7 @@ fn main() -> Result<()> {
     if let Some(args_database) = args.database {
         database = args_database;
         if backend == Backend::auto {
-            if database.starts_with("duckdb://") {
```

Comment:
I'd rather keep this as a reminder for other database backends.

Reply:
Sorry, I have deleted it as a result of modifying it as IDE said...